tests/no-relative-imports/allow-siblings/test.js.lint

```diff
@@ -0,0 +1,7 @@
+import OfficeApp from 'OfficeApp';
```

Comment:
Are these test file additions meant to be here? They seem unrelated, no?

Reply:
`tests` folder is demo of how TSLint can be used to test rules.

This is test for `no-relative-imports` rule which supports JS files too (as [mentioned](https://github.com/Microsoft/tslint-microsoft-contrib/pull/508#issuecomment-427144264) in previous PR)
For initial example I decided to include tests for both supported languages.

Should I remove `.js.lint` files?

Reply:
That seems fine. Just wanted to make sure this was intentional 😊